code/modules/mob/living/basic/heretic/fire_shark.dm

```diff
@@ -0,0 +1,52 @@
+/mob/living/basic/fire_shark
```

Comment:
I guess it is cheap to summon but an easily killed mob with a weak attack doesn't clearly offer very much to a heretic compared to existing minions.
Is there anything you could add to make it more interesting?